internal/pkg/flags/flag_to_value.go

```diff
@@ -22,19 +24,21 @@ func FlagToStringValue(cmd *cobra.Command, flag string) string {
 
 // Returns the flag's value as a bool.
 // Returns false if its value can not be converted to bool, or if the flag does not exist.
-func FlagToBoolValue(cmd *cobra.Command, flag string) bool {
+func FlagToBoolValue(cmd *cobra.Command, flag string, p *print.Printer) bool {
 	value, err := cmd.Flags().GetBool(flag)
 	if err != nil {
+		p.Debug(print.ErrorLevel, "convert flag to bool value err: %v", err)
```

Comment:
The flag string value could be added to the error message, but **only** if it is not shown in `err` already